api/src/main/java/uk/gov/hmcts/payment/api/configuration/security/SpringSecurityConfiguration.java

```diff
@@ -61,6 +61,7 @@ protected void configure(HttpSecurity http) throws Exception {
             .antMatchers(HttpMethod.GET, "/payments", "/api/**").permitAll()
             .antMatchers(HttpMethod.POST, "/api/**").permitAll()
             .antMatchers(HttpMethod.POST, "/payments/email-pay-reports").permitAll()
+            .antMatchers(HttpMethod.POST, "/payments/update").permitAll()
```

Comment:
do you aware of any backlog ticket to address security for this endpoint as it changes db state.

Reply:
The other job does not have security either as neither of them involves a risk of leaking anything

Reply:
security will be dealt with separately. please raise a jira ticket as tech debt though